src/include/common/vector/value_vector.h

```diff
@@ -149,43 +149,50 @@ struct KUZU_API BlobVector {
 class KUZU_API ListVector {
```

Comment:
Can you add a comment here to explain `ListVector` is used for both `VAR_LIST` and `ARRAY`?